i18n-msg-behavior.html

```diff
@@ -0,0 +1,224 @@
+<link rel="import" href="../polymer/polymer.html">
+<link rel="import" href="observer.html">
+
+<!--
+`<i18n-msg-behavior>` allows adding the i18n functionality via polymer behavior.
+-->
+<script>
+(function () {
+
+  window.I18nMsg = window.I18nMsg || {
+    lang: null,
+    url: 'locales',
+    locales: {}
+  };
+
+  var _instances = []; // Holds all i18n-msg elements to have them easily updated with translation messages.
+  var _fetching = false; // True if there's an outstanding XHR fetching a locale file.
+  var _numInstancesUpdated = 0;
+
+   /**
+    * `<i18n-msg-behavior>` allows adding the i18n functionality via polymer
+    * behavior.
+    *
+    * See the `<i18n-msg>` documentation for site configuration and language
+    * options.
+    *
+    * ### Example
+    *
+    * To use the `<i18n-msg-behavior>` add the behavior to the behaviors of the
+    * element and use the `i18n` property to add message ids and default values.
+    *
+    *     Polymer({
+    *       ...
+    *       behaviors: [i18nMsgBehavior],
+    *
+    *       properties: {
+    *         i18n: {
+    *           type: Object,
+    *           value: {
+    *             label: 'default label'
+    *               }
+    *         }
+    *       }
+    *        ...
+    *     });
+    *
+    * In the template you can then use those message ids:
+    *
+    *     <template>
+    *       <h1>[[i18n.label]]</h1>
+    *     </template>
+    *
+    * ### Custom events
+    *
+    * If you override the `ready` function then you will need to add the
+    * `_parentReady` call to the ready function to trigger the i18n:
+    *
+    *     ready: function() {
+    *       this._parentReady(window.I18nMsg);
```

Comment:
hmm. I wonder if there's a better name we could use, and one that's not private. `init`?